### ISSUE TITLE
Compact tree nodes do not have all accessibility attributes set

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#93465]
Issue Type: <b>Bug</b>

1. With NVDA running, Press sctrl+shift+e and navigate to a location that has the following structure: a/b/c.txt, where a and b are otherwise empty.

Expected: these should be exposed as treeitems with information about whether it is collapsed or expanded. However, these nodes are being exposed as links instead.

VS Code version: Code - Insiders 1.44.0-insider (e62c4302b0e4f3e28bbcb0f3b6e46bb00e39679c, 2020-03-26T05:51:24.240Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (4.80GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.1
gitlens|eam|10.2.1
prettier-vscode|esb|3.20.0
markdown-table-formatter|fcr|2.0.3
vscode-mysql|for|0.4.0
rest-client|hum|0.23.2
rust-analyzer|mat|0.2.100
Go|ms-|0.13.1
vscode-typescript-tslint-plugin|ms-|1.2.3
vsliveshare|ms-|1.0.1847
vscode-yaml|red|0.7.2
rust|rus|0.7.0
vscode-webhint|web|1.5.4

</details>

**Comment by isidorn** [comment]
@Neurrone great catch.
As a workaround I have fixed the role of those 'compact' elements as we call them. So they will now correctly have the role of `treeitem`.
However the correct fix would be that all tree attributes are nicely set on them. Which is `aria-level`, `aria-expended`, `aria-selected`, `aria-posinset`.

Until we get this nicely working I suggest as a workaround to disable compact elements
`"explorer.compactFolders": false`

Renaming this issue and also assigning to @joaomoreno

**Comment by isidorn** [comment]
Actualy duplicate of https://github.com/microsoft/vscode/issues/86414
